standalone_examples/lxd.py

```diff
@@ -0,0 +1,14 @@
+from pyinfra import host
+from pyinfra.modules import lxd
+
+SUDO = True
+
+# Note: This is an example of running a command on remote
+# init lxd using all defaults
+host.fact.command('lxd init --auto')
```

Comment:
This should really be:

```py
server.shell(
    {'Run lxd auto init'},
    'lxd init --auto',
)
```

The fact version will run before the actual deploy - facts should only ever read, not change anything.

Reply:
Thanks. Will fix.